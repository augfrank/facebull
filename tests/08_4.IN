M0   C0   C1  2103
M1   C0   C4  2260
M2   C0   C5  2874
M3   C0   C7  2447
M4   C1   C3  2866
M5   C1   C4  2125
M6   C1   C6  2021
M7   C1   C7  2999
M8   C2   C5  2327
M9   C3   C4  2354
M10   C3   C5  2740
M11   C3   C6  2473
M12   C3   C7  2060
M13   C4   C0  2702
M14   C4   C1  2355
M15   C4   C2  2591
M16   C4   C5  2311
M17   C4   C6  2260
M18   C5   C0  2825
M19   C5   C2  2058
M20   C5   C4  2864
M21   C6   C1  2448
M22   C6   C2  2551
M23   C6   C4  2270
M24   C6   C5  2309
M25   C6   C7  2114
M26   C7   C0  2605
M27   C7   C2  2398
M28   C7   C3  2975
