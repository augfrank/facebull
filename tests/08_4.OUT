19247
0 4 8 9 17 18 25 27
